pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -1,131 +1,114 @@
-schema = 1
+schema = 2
 
-[device]
-name = "lightning.qubit"
+[operators.gates.native]
 
-[operators]
-# Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "Projector",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-]
+CNOT                   = { properties = [ "invertible",                 "differentiable" ] }
```

Comment:
We could make this `controllable`, if that helps in any way.

Reply:
Make sense, but my intention for now is to describe state of the art here.

Reply:
Unless the CNOT C++ kernel accepts arbitrary _additional_ control wires it shouldn't be marked controllable, but if it does I don't see why we couldn't add it? @grwlf

Reply:
@dime10 CNOT is [absent](https://github.com/PennyLaneAI/pennylane-lightning/blob/37f19012eab034db981969c42e4c5705825fcb05/pennylane_lightning/core/src/gates/GateOperation.hpp#L73) from the Controlled enum. I believe it means it is not controllable.